src/packages/core/tests/server.test.ts

```diff
@@ -116,11 +116,7 @@ describe("server", () => {
     return deferred;
   }
 
-  // skip this test unless in GitHub Actions, as this test iterates over
-  // all available network interfaces and network interfaces on user
-  // machines are unpredictible and may behave in ways that we don't care
-  // about.
-  (process.env.GITHUB_ACTION ? describe : describe.skip)("listen", function () {
+  describe("listen", function () {
```

Comment:
This was a little confusing because there's a skip at the `describe` level as well as the test level, so I refactored to be just at the test level (even though we end up with a `describe` with no tests that run in a development environment).